File: CODE FILES/Backend/MyApiary/Resource.Api/Models/BeehiveUpdate.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace Resource.Api.Models
{
    public class BeehiveUpdate
    {
        [Required]
        public int Id { get; set; }

        [Required]
        public string Name { get; set; }

        [Required]
        public bool Alarm { get; set; }

        [Required]
        public int ApiariId { get; set; }
    }
}

File: CODE FILES/Mobile/MyApiary/app/src/main/java/projects/myapiary/models/BeehiveUpdate.kt
package projects.myapiary.models

import com.google.gson.annotations.SerializedName

data class BeehiveUpdate(
    @SerializedName("Id")
    val id: Int,
    @SerializedName("Name")
    val Name: String,
    @SerializedName("Alarm")
    val Alarm: Boolean,
    @SerializedName("ApiariId")
    val ApiariId: Int,
)
